### DISCUSSION TITLE
How to fine-tune using the model?

### DISCUSSION THREAD

**Discussion by Billy-Liu-DUT** [Kohulan/DECIMER-Image_Transformer#111] [Q&A] [answered]
I'd like to fine-tune this model. What should I do?

**Comment by Kohulan** [comment, marked as answer]
@Billy-Liu-DUT ,

Thank you for your interest in our DECIMER project. We provide the following resources for model training and implementation:

Training Script: Available in our GitHub repository at https://github.com/Kohulan/DECIMER-Image_Transformer/blob/master/DECIMER/DECIMER_EfficinetNetV2_Transfomer_Trainer.py
Model Checkpoints: Accessible via Zenodo at https://zenodo.org/records/8300489

For those interested in fine-tuning the model, we recommend consulting general transformer model fine-tuning tutorials and documentation, as the process follows standard practices. Currently, we do not have the capacity to write a detailed step-by-step approach to fine-tune the model.
